src/index.ts

```diff
@@ -141,10 +141,20 @@ function jigsawBinPath() {
 /**
  * Spawns a child process to run "jigsaw build" with the proper environment.
  */
-function spawnJigsawBuild(quiet = true): Promise<void> {
+function spawnJigsawBuild(hotFile: string, quiet = true): Promise<void> {
     return new Promise<void>((resolve, reject) => {
         const bin = jigsawBinPath();
         const envArg = process.env.NODE_ENV === 'development' ? 'local' : process.env.NODE_ENV;
+
+        // Remove the hot file when building for non-local environments
+        if (envArg !== "local" && existsSync(hotFile)) {
```

Comment:
thus this turns into this
```suggestion
        if (hotFile && existsSync(hotFile)) {
```

Reply:
Done!